addons/Points_To_Plot/src/presenter.js

```diff
@@ -68,11 +118,45 @@ function AddonPoints_To_Plot_create() {
             }
         });
     };
+
     presenter.setWorkMode = function() {
         var sourceModule = this.getSourceModule();
         sourceModule.removePointsStateMarks();
         sourceModule.enableUI(true);
     };
+
+    presenter.showAnswers = function() {
+        isShowAnswersActive = true;
+        var sourceModule = this.getSourceModule();
+        if (sourceModule == null) return;
+
+        sourceModule.setPtpShowAnswersPoints(presenter.showAnswersPoints);
+        if (presenter.showAnswersPoints.length > 0) {
+            sourceModule.ptpShowAnswers(presenter.showAnswersPoints.length - 1);
+        }
+    }
+
+    presenter.hideAnswers = function() {
+        if (!isShowAnswersActive) return;
+        isShowAnswersActive = false;
```

Comment:
imo ustawienie flagi na false powinno być za metodą ptpHideAnswers(), zwłaszcza, że jeżeli jakiś trafem nie będzie sourceModule to metoda ptpHideAnswers() się nie wykona